src/modules/sensors/rc_update.cpp

```diff
@@ -228,6 +228,23 @@ RCUpdate::rc_poll(const ParameterHandles &parameter_handles)
 			/* signal is lost or no enough channels */
 			signal_lost = true;
 
+			//This problem occurs at RC controllers which are use PPM signal on RFD 868+/900 Modems (Maybe it can be also related another types)
+
+		} else if (rc_input.channel_count ==
```

Comment:
You could limit this to PPM as well.

Reply:
I haven't personally used it, but that's my understanding.

![image](https://user-images.githubusercontent.com/84712/67022329-0f791200-f0cf-11e9-8180-74064150fa53.png)

Doing this directly in the PPM decode might be better.

https://github.com/PX4/Firmware/blob/2fcddd9b8d36b636e84f7e6c57f13904af79e430/platforms/nuttx/src/px4/stm/stm32_common/hrt/hrt.c#L447-L595

Reply:
I agree, it would be better doing that in the PPM decoder.

Reply:
But the change seems quite specific to fix exactly one corner case, therefore I think it's worth it.

Reply:
I'm not sure what you mean. It's a corner case that applies to one specific PPM receiver.

Reply:
Yes but it's good for that one corner case :smile:.

Reply:
My take is that this is good for this corner case, and it doesn't cause any harm so it's worthwhile to be merged.